integtest/spec/single_book_spec.rb

```diff
@@ -0,0 +1,209 @@
+# frozen_string_literal: true
+
+RSpec.describe 'building a single book' do
+  ##
+  # Builds a path to afile in the source.
+  def source_file(file)
```

Comment:
Would these helper methods be useful for other tests? If so, it might be worth making a `support` directory in the `spec` directory so other spec files can use it.

Reply:
Probably! I put the ones that I 100% sure would be shared in spec_helper and the ones I wasn't 100% sure of I left here and figured I'd move when they turned out they were helpful.